www/docs/en/dev/guide/platforms/android/plugin.md

```diff
@@ -118,7 +118,8 @@ exception names as much as possible.
 The plugin's JavaScript does _not_ run in the main thread of the
 `WebView` interface; instead, it runs on the `WebCore` thread, as
 does the `execute` method.  If you need to interact with the user
-interface, you should use the following variation:
+interface, you should use the [Activity's `runOnUiThread`][ref-runonuithread]
```

Comment:
This appears very weird on the github rich diff. I hope it's better on the website?